google_analytics/preprocessing.py

```diff
@@ -20,7 +20,10 @@ def load(path, nrows=None):
 
     # Normalize JSON columns
     for column in JSON_COLUMNS:
-        column_as_df = pd.io.json.json_normalize(df[column])
+        if format_solver:
```

Comment:
What excatly the `format_solver` does and when should I use it?

Reply:
In some laptops, the command  `pd.io.json.json_normalize` does not work to change the format to JSON. A way to solve it is to apply the format to every row as is proposed on the format solver.